frontend/userExtensions/views.py

```diff
@@ -71,10 +71,18 @@ def login_view(request, *args, **kwargs):
     body_unicode = request.body.decode('utf-8')
     data = json.loads(body_unicode)
    
-    username = data.get("username_mail")
+    username_email = data.get("username_email")
     password = data.get("password")
 
-    user = auth.authenticate(username=username, password=password)
+    if "@" in username_email:
```

Comment:
If you want to check if it's mail, the @ sign should be disabled in the name, just a reminder.

Reply:
I'll fix it, thx.